test/utils/cyaHelpers/item/ItemAnswersHelperSpec.scala

```diff
@@ -0,0 +1,547 @@
+ * limitations under the License.
+ */
+
+package utils.cyaHelpers.item
+
+import base.SpecBase
+import controllers.item.routes._
+import controllers.item.dangerousGoods.index.routes.UNNumberController
+import generators.Generators
+import models.reference.Country
+import models.{DeclarationType, Mode}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import pages.item._
+import pages.item.dangerousGoods.index.UNNumberPage
+
+class ItemAnswersHelperSpec extends SpecBase with ScalaCheckPropertyChecks with Generators {
+
+  "ItemAnswersHelper" - {
+
+    "itemDescription" - {
+      "must return None" - {
+        "when DescriptionPage is undefined" in {
+          forAll(arbitrary[Mode]) {
+            mode =>
+              val helper = new ItemAnswersHelper(emptyUserAnswers, mode, itemIndex)
+              val result = helper.itemDescription
+              result mustBe None
+          }
+        }
+      }
+
+      "must return Some(Row)" - {
+        "when DescriptionPage is defined" in {
+          forAll(arbitrary[Mode], nonEmptyString) {
+            (mode, description) =>
+              val answers = emptyUserAnswers
+                .setValue(DescriptionPage(itemIndex), description)
+
+              val helper = new ItemAnswersHelper(answers, mode, itemIndex)
+              val result = helper.itemDescription.get
+
+              result.key.value mustBe "Item description"
+              result.value.value mustBe description
+
+              val actions = result.actions.get.items
+              actions.size mustBe 1
+              val action = actions.head
+              action.content.value mustBe "Change"
+              action.href mustBe DescriptionController.onPageLoad(answers.lrn, mode, itemIndex).url
+              action.visuallyHiddenText.get mustBe "item description"
+              action.id mustBe "change-description"
+          }
+        }
+      }
+    }
+
+    "declarationType" - {
+      "must return None" - {
+        "when DeclarationTypePage is undefined" in {
+          forAll(arbitrary[Mode]) {
+            mode =>
+              val helper = new ItemAnswersHelper(emptyUserAnswers, mode, itemIndex)
+              val result = helper.declarationType
+              result mustBe None
+          }
+        }
+      }
+
+      "must return Some(Row)" - {
+        "when DeclarationTypePage is defined" in {
+          forAll(arbitrary[Mode], arbitrary[DeclarationType]) {
+            (mode, declarationType) =>
+              val answers = emptyUserAnswers
+                .setValue(DeclarationTypePage(itemIndex), declarationType)
+
+              val helper = new ItemAnswersHelper(answers, mode, itemIndex)
+              val result = helper.declarationType.get
+
+              result.key.value mustBe "Item declaration type"
+              result.value.value mustBe messages(s"${DeclarationType.messageKeyPrefix}.$declarationType")
```

Comment:
Minor suggestion but fir the enum methods it's not a bad check to see that the message definitely exists for that `s"${DeclarationType.messageKeyPrefix}.$declarationType"` key. Something like:

https://github.com/hmrc/manage-transit-movements-arrival-frontend/blob/841b9d22d2299cf4d9f5dc3969588fab85513ed1/test/utils/LocationOfGoodsAnswersHelperSpec.scala#L56-L58